test/rc-test.js

```diff
@@ -30,8 +30,8 @@ test('custom config and aliases', function (t) {
     t.equal(rc.version, rc.v, 'version alias')
     t.equal(rc.help, true, 'help is set')
     t.equal(rc.help, rc.h, 'help alias')
-    t.equal(rc.path, '../some/other/path', 'correct path')
-    t.equal(rc.path, rc.p, 'path alias')
+    t.equal(rc.path, path.resolve(tmp, '../some/other/path'), 'correct path')
+    t.equal(rc.path, path.resolve(tmp, rc.p), 'path alias')
```

Comment:
```suggestion
```

The second test has become meaningless, because `rc.p` now holds the original path (if any) while `rc.path` holds the absolute path. The test used to assert that `-p` is a command line alias for `--path`, which IMO we don't need to test because it's a behavior provided by the `rc` module. And because we don't reference `rc.p` anywhere else, it's also OK that `rc.p` no longer equals `rc.path`.